runtime/lua/json.lua

```diff
@@ -0,0 +1,383 @@
+--
+-- json.lua
```

Comment:
Not random times, just times the event loop is invoked. As previously this could be done in a way that queues all vimL until later, such spots could potentially be vimL GC unsafe. Can container allocation invoke GC or does that only happen at states? 

The plan is indeed to improve jobs so they can be used here.